code/game/objects/items/toys.dm

```diff
@@ -1495,3 +1495,44 @@
 	if(cooldown <= world.time)
 		cooldown = (world.time + 10)
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, src, 'sound/effects/blobattack.ogg', 50, FALSE), 0.5 SECONDS)
+
+
+/*
+ * Eldrich Toys
+ */
+
+/obj/item/toy/eldrich_book
```

Comment:
i don't know if there's a joke i'm missing but i'm pretty sure it's **eldritch** not eldrich

Reply:
typo